backends/intel_gpu/README_cn.md

```diff
@@ -0,0 +1,60 @@
+# 飞桨自定义接入硬件后端(Custom CPU)
```

Comment:
Please delete Chinese readme file if not applicable

Reply:
deleted